### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader:VoiceOver does not read the ordering for list items under “Database the external table will be created  in ” combo box  .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5971]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Tools Name: VoiceOver
MAS References: MAS1.3.1

**Steps to Reproduce:**
1.Install the app Azure Data Studio.
2.Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files.
3.Start VoiceOver.
4.Navigate to the “Database the external table will be created  in ” combo box .
5.Observe that screen reader is not read the ordering for list items under  “Database the external table will be created  in ” combo box.

**Actual:**
Navigate till “Database the external table will be created  in ” combo box ,screen reader is not read the ordering for list items under  “Database the external table will be created  in ” combo box.
**Similar issue has been found under the below pages:**
1.**MapDataSrcObjects**- Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
**Expected:**
Screen Reader should read order for the list item for example for item "WizardTesting" screen reader should announce "WizardTesting 1 of 5 level 1" and similar for other items.
Similar issue is also observed for other Combo boxes.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader user is unable to understand order of checkboxes are present on the page.

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[5971_[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3275321/5971_.Accessibility.A11y_AzureDataStudio_CSVSelectDestinationDatbse_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Possibly related to #5969

**Comment by corivera** [comment]
VoiceOver doesn't seem to read out ordering at all. I created a simple html file with a select box, and it didn't announce any ordering like "1 of 15", as would be heard with Narrator, just the labels of the dropdown items.
